core/modules/user/user.admin.inc

```diff
@@ -288,16 +288,22 @@ function user_admin_settings($form, &$form_state) {
   $form['email'] = array(
     '#type' => 'vertical_tabs',
   );
-  // These email tokens are shared for all settings, so just define
-  // the list once to help ensure they stay in sync.
-  $email_token_help = t('Available variables are: [site:name], [site:url], [user:name], [user:mail], [site:login-url], [site:url-brief], [user:edit-url], [user:one-time-login-url], [user:cancel-url].');
+
+  // Add token JavaScript to the form.
+  $form['#attached']['js'][] = backdrop_get_path('module', 'system') . '/js/token.js';
```

Comment:
I don't think this JS file needs to be added here. The JS file will be added when the dialog is opened.